src/flows/pipes/Markdown/MarkdownPanel.tsx

```diff
@@ -6,6 +6,7 @@ import {
   TechnoSpinner,
 } from '@influxdata/clockface'
 import classnames from 'classnames'
+import {debounce} from 'lodash'
```

Comment:
ewwwwwwwwww lodash what the heck mang

Reply:
I mean...ok, I can implement it myself but we do already have the package installed so I figured what's the harm 🤷

Reply:
debounce removed! It's unneeded to fix this issue